app/views/exam_templates/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% content_for :title, ExamTemplate.model_name.human.pluralize %>
+
+<%= render partial: 'form',
+           locals: { exam_template: @exam_templates }%>
```

Comment:
Do `@exam_templates[0]` here for now. Eventually you'll want a separate form for each exam template.